test/meanvar.cpp

```diff
@@ -178,20 +192,20 @@ meanvar_test_gen(string name, int in_index, int weight_index, af_var_bias bias,
   }
   if(weight_index == -1) {
       return meanvar_test<T> (name,
-                              inputs[in_index],
-                              empty,
+                              move(inputs[in_index]),
```

Comment:
I don't understand why inputs has to be a vector of af_array when there is only one af_array in use in all cases. Am I missing to see something ?

Reply:
Even `outputs` for that matter I think. I can see why `tests_` has to be vector of vectors but not why `outputs` has to be a vector of vectors.

Reply:
It makes sense with the for loop back now.